config/src/config/benchmark.rs

```diff
@@ -0,0 +1,40 @@
+// Copyright © Aptos Foundation
+// SPDX-License-Identifier: Apache-2.0
+
+use serde::{Deserialize, Serialize};
+
+#[derive(Clone, Copy, Debug, Deserialize, PartialEq, Eq, Serialize)]
+#[serde(default, deny_unknown_fields)]
+pub struct BenchmarkConfig {
```

Comment:
nit: maybe `NetworkBenchmarkConfig`? Also, should this live in the `network_config.rs` file?